medcat/config_meta_cat.py

```diff
@@ -48,11 +47,19 @@ class Config:
 
 class Model(MixingConfig, BaseModel):
     """The model part of the metaCAT config"""
-    model_name: str = 'lstm'
+    model_name: str = 'bert'
```

Comment:
Changing the default here could change existing behaviour. We want to avoid that.

Perhaps a separate method to get a default config suitable for BERT rather than LSTM?

Reply:
I've reverted the change